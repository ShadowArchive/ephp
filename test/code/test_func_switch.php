<?php

function translate_level_to_role($level) {
        switch ($level) {
        case 10:
        case 9:
        case 8:
                return 'administrator';
        case 7:
        case 6:
        case 5:
                return 'editor';
        case 4:
        case 3:
        case 2:
                return 'author';
        case 1:
                return 'contributor';
        case 0:
                return 'subscriber';
        }
}

for ($i=0; $i<10; $i++)
    print "$i -> " . translate_level_to_role($i) . "\n";

