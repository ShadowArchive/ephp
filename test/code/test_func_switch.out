0 -> subscriber
1 -> contributor
2 -> author
3 -> author
4 -> author
5 -> editor
6 -> editor
7 -> editor
8 -> administrator
9 -> administrator
